Tests/RockLib.Configuration.Conditional.Tests/ConditionalConfigurationSectionTests.cs

```diff
@@ -0,0 +1,365 @@
+using System;
+using System.Collections.Generic;
+using FluentAssertions;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Xunit;
+
+namespace RockLib.Configuration.Conditional.Tests;
+
+public class ConditionalConfigurationSectionTests

```

Comment:
We typically make our test classes `static`.

Reply:
Updated.